examples/AsyncTranscribeMediaUrl.java

```diff
@@ -0,0 +1,102 @@
+package ai.rev.speechtotext;
+
+import ai.rev.speechtotext.models.asynchronous.RevAiCaptionType;
+import ai.rev.speechtotext.models.asynchronous.RevAiJob;
+import ai.rev.speechtotext.models.asynchronous.RevAiJobOptions;
+import ai.rev.speechtotext.models.asynchronous.RevAiJobStatus;
+import ai.rev.speechtotext.models.asynchronous.RevAiTranscript;
+import ai.rev.speechtotext.models.vocabulary.CustomVocabulary;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+public class AsyncTranscribeMediaUrl {
+
+  public static void main(String[] args) {
+    // Assign your access token to a String
+    String accessToken = "your_access_token";
+
+    // Initialize the ApiClient with your access token
+    ApiClient apiClient = new ApiClient(accessToken);
+
+    // Create a custom vocabulary for your submission
+    CustomVocabulary customVocabulary =
+        new CustomVocabulary(Arrays.asList("Robert Berwick", "Noam Chomsky", "Evelina Fedorenko"));
+
+    // Initialize the RevAiJobOptions object and assign
+    RevAiJobOptions revAiJobOptions = new RevAiJobOptions();
+    revAiJobOptions.setCustomVocabularies(Arrays.asList(customVocabulary));
+    revAiJobOptions.setMetadata("My first submission");
+    revAiJobOptions.setCallbackUrl("https://example.com");
+    revAiJobOptions.setSkipPunctuation(false);
+    revAiJobOptions.setSkipDiarization(false);
+    revAiJobOptions.setFilterProfanity(true);
+    revAiJobOptions.setSpeakerChannelsCount(null);
+
+    RevAiJob submittedJob;
+
+    String mediaUrl = "https://your.media.url";
```

Comment:
You can use the example media url we use in other places for rev.ai. It contains a real media file.
`https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3`